clj/src/cljd/core.cljd

```diff
@@ -8246,4 +8246,51 @@
       (doseq [tap @tapset]
         (try
           (tap x)
-          (catch Exception ex))))))
\ No newline at end of file
+          (catch Exception ex))))))
+
+(defprotocol Inst
+  (inst-ms* [inst]))
+
+(extend-type DateTime
+  IEquiv
+  (-equiv [o other]
+    (and (dart/is? other DateTime)
+         (== (.-millisecond o) (.-millisecond ^DateTime other))))
```

Comment:
why do cljs rely on `millisecond`. @cgrand do you know if there are historical reason? The `==` operator in Dart seems good to me https://api.flutter.dev/flutter/dart-core/DateTime/operator_equals.html

Reply:
Date objects in JS use identity for equality, so they chose to compare milliseconds to get equality. Like @dupuchba said we don't need this extend-type in ClojureDart because DateTime equality is already correct.